### PR TITLE
add logger.py

### PR DESCRIPTION
**Pull request by Junefen**
Define the logger handler for project using.

### CODE REVIEW

File: virtwho/logger.py [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+        Define the log path, log file, log level, log formatter.
+        The default log directory is logs.
+        """
+        self.logger = logging.getLogger(logger)
+        self.logger.setLevel(logging.DEBUG)
+        self.logger.handlers = []
+        self.log_path = os.path.join(
+            os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
+            'logs')
+        if not os.path.exists(self.log_path):os.mkdir(self.log_path)
```
Comment (eko999) [inline virtwho/logger.py:29]: It's my fault, it's not pythonic, please change it to:
```
if not os.path.exists(self.log_path):
    os.mkdir(self.log_path)
```

File: virtwho/logger.py [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
-"""
+import os
+import time
+import logging
+
+
+class Logger:
+    """
+    Usage:
+        from hypervisor import log
```
[the remark this answers is not in the document]
Comment (Junefen) [reply virtwho/logger.py]: Done

File: virtwho/logger.py (hunk 1 above)
Comment (Junefen) [reply virtwho/logger.py]: Done

**Review by hkx303 — APPROVED** [review]
LGTM, ACK

**Pull request merged**